.buildkite/Brewfile

```diff
@@ -0,0 +1,29 @@
+# This Brewfile is for setting up and managing the small amount of
+# Homebrew-related infrastructure we need. As our MacOS build process
+# matures, we can whittle this down.
+
+# Though our builders will already have this installed, we can take
+# advantage of the Brewfile to ensure that we've got the latest
+# version of hab installed. There are other ways to do this, of
+# course, but none that take 2 lines of code.
+tap "habitat-sh/habitat"
+brew "hab"
+
+# Our builders currently have Bash 3.2.57 installed, but our Habitat
+# package build process requires a more modern version. The Linux
+# build currently uses Bash 4.3.42, and Homebrew currently brings in
+# 4.4.23.
+#
+# Until our builders have a modern Bash installed, we'll just bring our
+# own.
+brew "bash"
```

Comment:
We can assume that the habitat omnibus package has current bash in it in the future.

Reply:
![tenor-71154006](https://user-images.githubusercontent.com/207178/43410407-c209199e-93f4-11e8-8463-2079b58188d8.gif)